limitador-server/src/envoy_rls/envoy_types.rs

```diff
@@ -4,8 +4,9 @@ pub mod envoy {
     pub mod config {
         #[path = "."]
         pub mod core {
-            #[path = "envoy.config.core.v3.rs"]
-            pub mod v3;
+            pub mod v3 {
+                include!(concat!(env!("OUT_DIR"), "/envoy.config.core.v3.rs"));
```

Comment:
We switched from `include!` to `path` because @unleashed mentioned that `include!` does not work in some IDEs https://github.com/3scale-labs/limitador/pull/5#discussion_r534291152